CSS/module.css

```diff
@@ -0,0 +1,154 @@
+/*reusable styles*/
+
+.button {
+  border: none;
+  font-family: Open Sans Condensed;
+  font-size: 15px;
+}
+
+.paragraphText {
```

Comment:
Sorry I didn't catch this on the first go-through... just pointing it out, but not dinging you on the points for it..

Since this seems to be the main paragraph formatting throughout the entire page, it might be more suitable as the generic p tag styling. Then you can just override it anywhere it needs to be different. It will save you from excessive classing of your html.